### PR TITLE
Replace `Change Password` with `Dashboard` on header #38

### PR DESCRIPTION
**Pull request by SwarupKharul**

### CODE REVIEW

File: templates/snippets/header.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {% comment %}
 <div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 bg-dark border-bottom shadow-sm sticky-top">
-    <h2 class="mb-3 my-md-0 mr-md-auto font-weight-normal" >
-        <a class="text-decoration-none" href="{% url 'home:index' %}" style="color:#ff8c00;">Jagrati</a>								<!--Change this-->
+    <h2 class="mb-3 my-md-0 mr-md-auto font-weight-normal">
+        <a class="text-decoration-none" href="{% url 'home:index' %}" style="color:#ff8c00;">Jagrati</a>
+        <!--Change this-->
```
Comment (garg3133) [inline templates/snippets/header.html:5]: Why have you added this comment?

File: templates/snippets/header.html [hunk 2]
```diff
@@ -49,25 +51,27 @@ <h2 class="my-0 mr-md-auto font-weight-normal" >
...
             <li class="nav-item">
                 <a class="btn btn-outline-danger" href="{% url 'accounts:logout' %}">Sign Out</a>
             </li>
             {% endif %}
-            
+
             {% else %}
             <li class="nav-item">
                 <a class="btn btn-outline-danger" href="{% url 'accounts:login_signup' %}">Sign In</a>
             </li>
```
Comment (garg3133) [inline templates/snippets/header.html:73]: Can you also add a link here to the home page, just like the dashboard link above, which will be displayed if the user is not logged in? (Dashboard link is displayed only if the user is logged in.)

Comment (SwarupKharul) [reply templates/snippets/header.html:73]: ok, I will do it

File: templates/snippets/header.html (hunk 1 above)
Comment (SwarupKharul) [reply templates/snippets/header.html:5]: Actually I have an extension in VSCode which automatically beautifies the code. Should I make it in the way as it was before? I have not added anything, only the formatting is changed

Comment (garg3133) [reply templates/snippets/header.html:5]: The new formatting is fine, just remove the `<!--Change this-->` comment.

What's the extension btw?

Comment (SwarupKharul) [reply templates/snippets/header.html:5]: Simple React Snippets

**Pull request closed without merging**